Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.MigrationsTool/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace MvvmSeed.MigrationsTool
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //NOTES: We need this console projects to generate migrations since EF Tools are not available for the other platforms yet
+            //TODO: Do something cool here to generate migrations automatically
+            Console.WriteLine("WIP");
```

Comment:
By manualIly I meant opening a console and running commands using `dotnet ef migrations`.

The problem is `EntityFrameworkCore.Tools.DotNet` cannot be installed on a .NET Standard project (yet). I would like to review this process to minimize the steps or at least have a detailes document on how to do it.